### PR TITLE
Add e2e tests for VIC

### PR DESCRIPTION
**Pull request by jbalboni**
These are e2e tests up to the photo tool. I'll create an issue to finish the rest once the form is more complete.

### CODE REVIEW

File: test/vic-v2/00-all-fields.e2e.spec.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+const E2eHelpers = require('../e2e/e2e-helpers');
+const Timeouts = require('../e2e/timeouts');
+const PageHelpers = require('../e2e/vic-helpers');
+const testData = require('./schema/maximal-test.json');
+
+const runTest = E2eHelpers.createE2eTest(
+  (client) => {
```
Comment (rianfowler) [inline test/vic-v2/00-all-fields.e2e.spec.js]: What's `client`?

Comment (jbalboni) [reply test/vic-v2/00-all-fields.e2e.spec.js]: It's the interface for Nightwatch, the browser testing library we use: http://nightwatchjs.org/

File: test/e2e/vic-helpers.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
+const mock = require('./mock-helpers');
+
+function completeApplicantInformation(client, data) {
+  client
+    .fillName('root_veteranFullName', data.veteranFullName)
+    .selectRadio('root_gender', data.gender)
+    .fill('input[name="root_veteranSocialSecurityNumber"]', data.veteranSocialSecurityNumber)
+    .fillDate('root_veteranDateOfBirth', data.veteranDateOfBirth)
```
Comment (cehsu) [inline test/e2e/vic-helpers.js]: Unrelated to these changes, it looks like we're not validating this field like we do in some other forms (such as HCA), should we be checking for a minimum age?

**Review by cehsu — APPROVED** [review]
LGTM. I had a question unrelated to these changes.

File: test/e2e/vic-helpers.js (hunk 2 above)
Comment (jbalboni) [reply test/e2e/vic-helpers.js]: Yeah, might be worth doing.

**Pull request merged**